.github/workflows/test.yml

```
@@ -39,7 +39,7 @@ jobs:
         uses: actions/cache@v3
         with:
           path: /tmp/nix-cache
-          key: ${{ runner.os }}-flake-${{ hashFiles('*.lock') }}
+          key: ${{ runner.os }}-flake-${{ hashFiles('*.lock') }}-temp
```

```suggestion
          key: ${{ runner.os }}-flake-${{ hashFiles('*.lock') }}
```
Just needed to work with my own cache to make sure this PR is working right. Need to clear all the caches right before this is queued for merge.